fhir/resource_structure_dev/encounter.json

```diff
@@ -0,0 +1,53 @@
+{
+  "resourceType" : "Encounter",
+  "id" : "encounter_example_hospital_admission_for_covid",
+  "meta" : {
+    "profile" : [
+      "our_url_here"
+    ]
+  },
+  "extension" : [
+    {
+      "url" : "our-research-study-url",
+      "valueReference": {"reference": "ResearchStudy/<example-research-study>"}
+    }
+  ],
+  "subject": {"reference": "Patient/<id>"},
+  "status": "completed",
+  "class" : [{
+    "coding" : [{
+      "system" : "http://terminology.hl7.org/CodeSystem/v3-ActCode",
+      "code" : "IMP",
+      "display" : "inpatient encounter"
+    }]
```

Comment:
type and serviceType could also be used to classify the encounter type. type = snomed coding. Not sure of the value of multiple classifications however